metadata-ingestion/src/datahub/ingestion/source/schema_inference/parquet.py

```diff
@@ -81,14 +115,23 @@ def infer_schema(self, file: IO[bytes]) -> List[SchemaField]:
 
         fields: List[SchemaField] = []
 
+        meta_data_fields = parse_metadata(
+            schema.metadata[b"org.apache.spark.sql.parquet.row.metadata"]
+        )
```

Comment:
Is there a guarantee that this metadata field will always exist? We should consider treating it as optional.